### PR TITLE
Make the current top speed clickable

### PR DESCRIPTION
**Pull request by simoexpo**

### CODE REVIEW

File: src/components/live-traffic.vue [hunk 1]
```diff
@@ -95,11 +105,13 @@ export default class extends DashboardWidget {
     this.BoxedCountFlight = Object.keys(flights).length;
     this.BoxedCountAirline = airlines.size;
     if (this.BoxedCountAirline !== 0) {
-      this.BoxedMaxSpeed = Math.max.apply(
-        0,
-        flights.map((f: Flight) => f.speed),
-      );
+      const maxSpeedFlight = flights.reduce((max: Flight, f: Flight) => {
+        return ( f.speed > max.speed ? f : max );
```
Comment (massimosiani) [inline src/components/live-traffic.vue]: fails il flights is empty?
also, you don't need return here (nor the braces)

File: src/libs/map-engine.ts [hunk 2]
```diff
@@ -2,6 +2,7 @@ import WorldControl from '../mapbox-controls/world-control';
 import { Flight, AirportInfo } from '../interfaces/flight';
 import BoundingBox from './bounding-box';
 import {} from 'googlemaps';
+import { mapState } from 'vuex';
```
Comment (massimosiani) [inline src/libs/map-engine.ts]: no longer necessary?

File: src/components/live-traffic.vue (hunk 1 above)
Comment (simoexpo) [reply src/components/live-traffic.vue]: the if condition should fail if flights is empty, thanks for return (and braces) 😬

File: src/libs/map-engine.ts (hunk 2 above)
Comment (simoexpo) [reply src/libs/map-engine.ts]: no idea why I added this :|

File: src/components/live-traffic.vue [hunk 3]
```diff
@@ -105,9 +105,7 @@ export default class extends DashboardWidget {
     this.BoxedCountFlight = Object.keys(flights).length;
     this.BoxedCountAirline = airlines.size;
     if (this.BoxedCountAirline !== 0) {
-      const maxSpeedFlight = flights.reduce((max: Flight, f: Flight) => {
-        return ( f.speed > max.speed ? f : max );
-      });
+      const maxSpeedFlight = flights.reduce((max: Flight, f: Flight) => f.speed > max.speed ? f : max);
```
Comment (massimosiani) [inline src/components/live-traffic.vue]: I mean, if flights is an empty list, this throws an exception. Maybe we can define a default in this case and pass it as the second argument of reduce.
The default may be `{ speed: 0, icaoNumber: undefined }`.

Comment (massimosiani) [reply src/components/live-traffic.vue]: Or, we can check the flights.size > 0 above.

**Pull request merged**